flexmeasures/data/queries/generic_assets.py

```diff
@@ -12,9 +15,9 @@
 
 
 def query_assets_by_type(
-    type_names: Union[List[str], str],
-    account_id: Optional[int] = None,
-    query: Optional[Query] = None,
+    type_names: list[str] | str,
+    account_id: int | None = None,
+    query: Query | None = None,
```

Comment:
Is this still a `Query` object? Likewise for the return value.

Reply:
No, It's Select now.